### PR TITLE
add GSI-13 manifests and update AOSP revision

### PR DESCRIPTION
**Pull request by ittat**

### CODE REVIEW

**Comment by ittat** [comment]
with this pull request, I did two things:
 1 : updated aosp-13 build version from r6 to r24
 2 : added gsi-13 build manifests
 
 I uploaded the new images fille in [here](https://github.com/capyloon-gsi/B2G/releases/tag/r24) including [bluejay.tar.xz](https://github.com/capyloon-gsi/B2G/releases/download/r24/bluejay.tar.xz) and [gsi_system.tar.xz](https://github.com/capyloon-gsi/B2G/releases/download/r24/gsi_system.tar.xz).
 
 I tested GSI work fine on Oneplus 3t  and Duoqin 2 pro.

File: base-13.xml [hunk 1]
```diff
@@ -29,12 +17,25 @@
...
     <linkfile src="root.bp" dest="Android.bp"/>
     <linkfile src="bootstrap.bash" dest="bootstrap.bash"/>
   </project>
-  <project name="platform_system_core" path="system/core" remote="capyloon" revision="gonk_android-13.0.0_r6"/>
-  <project name="platform_system_vold" path="system/vold" remote="capyloon" revision="gonk_android-13.0.0_r6"/>
-  <project name="platform_system_connectivity_wificond" path="system/connectivity/wificond" remote="capyloon" revision="gonk_android-13.0.0_r6"/>
+  <project name="platform_system_core" path="system/core" remote="capyloon" revision="gonk_android-13.0.0_r24"/>
+  <project name="platform_system_vold" path="system/vold" remote="capyloon" revision="gonk_android-13.0.0_r24"/>
+  <project name="platform_system_connectivity_wificond" path="system/connectivity/wificond" remote="capyloon" revision="gonk_android-13.0.0_r24"/>
+  <project name="platform_system_sepolicy" path="system/sepolicy" remote="capyloon" revision="gonk_android-13.0.0_r24"/>
```
Comment (ittat) [inline base-13.xml:27]: I want to push all "gonk_android-13.0.0_r24"  branch from temporary "capyloon-gsi" organization back to the "capyloon" organization, but I find that I don't have permission to create a new branch.

File: gsi-13.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_frameworks_opt_telephony" path="frameworks/opt/telephony" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_netd" path="system/netd" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_nfc" path="system/nfc" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_bpf" path="system/bpf" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_bionic" path="bionic" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_system_linkerconfig" path="system/linkerconfig" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_hardware_interfaces" path="hardware/interfaces" />
+  <project groups="gsi"  remote="capyloon-gsi" revision="android-13.0.0_r24-td" name="platform_frameworks_libs_net" path="frameworks/libs/net" />
```
Comment (ittat) [inline gsi-13.xml]: There is another problem: these repos  added phh`s patches, which are compatible with more non-standard devices running GSI. Due to lack of time, I couldn't weed out patches that weren't necessary for B2G. So many repositories, I'm not sure there's a need to fork back to Capyloon.

**Comment by ittat** [comment]
Everything is done and ready to merge

**Comment by fabricedesre** [comment]
> with this pull request, I did two things: 1 : updated aosp-13 build version from r6 to r24 2 : added gsi-13 build manifests
> 
> I uploaded the new images fille in [here](https://github.com/capyloon-gsi/B2G/releases/tag/r24) including [bluejay.tar.xz](https://github.com/capyloon-gsi/B2G/releases/download/r24/bluejay.tar.xz) and [gsi_system.tar.xz](https://github.com/capyloon-gsi/B2G/releases/download/r24/gsi_system.tar.xz).
> 
> I tested GSI work fine on Oneplus 3t and Duoqin 2 pro.

I tested the bluejay image you uploaded and it works fine too.

File: gsi-13.xml (hunk 2 above)
Comment (fabricedesre) [reply gsi-13.xml]: We need to be careful with the ones that we already forked for b2g-specific patches. Apart from that, it's a bit annoying but I'm not opposed to host forks that are GSI specific.

Comment (ittat) [reply gsi-13.xml]: Yes, I checked out those b2g related forks to make sure that the GSI patches doesn't affect the work of b2g

**Comment by ittat** [comment]
Merged,have a nice weekend!

**Pull request merged**